theme/less/_content.less

```diff
@@ -1,4 +1,62 @@
-main article img
+article
```

Comment:
meeting-notes:
- Bold fonts seem to be too bold in comparison to Drupal, we can try using 600 semi-bold instead as a compromise.
- And let's try move content up, if possible.

Reply:
This I think is done.